website/docs/reference/datasources/airtable.md

```diff
@@ -4,399 +4,334 @@ sidebar_position: 1
+**Setup**: create your query called `DeleteUser` based on your Airtable datasource. This query should use the **Delete A Record** command. Enter the appropriate values for **Base ID** and **Table Name**. Finally, You should have a [Table widget](/reference/widgets/table) `UsersTable` containing your spreadsheet data from a [**List Records**](#list-records) query.
 
-#### Record ID
+Create a [Button widget](/reference/widgets/button) on the canvas and update its **Label** to "Delete." Set its **onClick** event to execute your `DeleteUser` query:
 
-Within Airtable, each record has a unique identifier known as a Record ID. If you are familiar with Entity-Relationship Diagrams or ERDs, then the record id would be the primary key. For most, record IDs remain behind the scenes.
+```javascript
+// in the Delete button's onClick event
+{{ DeleteUser.run() }}
+```
 
-After expanding a record, take notice of your web address bar. The URL will be as below**:**
+To delete a record, you need only provide the `id` of the record to delete. In your `DeleteUser` query's **Record ID** field, bind the value of the selected row of the `UsersTable`:
 
-```
-airtable.com/appxxxxxxxxx/tblxxxxxxxxx/viwxxxxxxxx/recxxxxxxxx
+```javascript
+// DeleteUser query's Row Index field
+{{ UsersTable.selectedRow.id }}
 ```
 
-The `recxxxxxxx` portion of the URL string is the unique record id.
+Now when the button is clicked, the query is run and the corresponding row is deleted from your Airtable base.
 
-```
-Example URL - https://airtable.com/appUgxxxxx/tblxZhrixxxxx/viwGM7xxxxx/recw3xxxxx?blocks=hide
+## Commands
 
-appUgxxxxx- Base ID
-tblxZhrixxxxx - Table ID
-viwGM7xxxxx - View ID
-Recw3xxxxx - Record ID
-```
+**Command** sets the type of action you want to perform with your query.
 
-:::info
-Use the [List Records command](./airtable.md#list-records) to get the record ID for each record.
-:::
+| **Command**           | **Description**                                                                    |
+| ----------------------  | ---------------------------------------------------------------------------------- |
+| **List Records**        | Fetch records from a base table.                                                   |
+| **Create Records**      | Add new records to a base table.                                                   |
+| **Delete A Record**     | Delete a record from a base table by its Record ID.                                |
+| **Retrieve A Record**   | Fetch a single record by its Record ID.                                            |
+| **Update Records**      | Update existing records in a base table, referenced by their Record ID.            |
+
+## Troubleshooting
```

Comment:
If there are no troubleshooting topics related to Airtable remove this section entirely